app/Http/Controllers/FAQController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\FAQ;
+use HTMLPurifier;
+use HTMLPurifier_Config;
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+
+class FAQController extends Controller
+{
+    protected HTMLPurifier $purifier;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $config = HTMLPurifier_Config::createDefault();
+        $this->purifier = new HTMLPurifier($config);
+    }
+
+    /**
+     * @return View
+     */
+    public function index()
+    {
+        return view("faq.index", ["faqs" => FAQ::all()]);
+    }
+
+    /**
+     *
+     * @param Request $request
+     * @return View
+     */
+    public function create(Request $request)
+    {
+        return view("faq.create", ["request" => $request]);
+    }
+
+    /**
+     *
+     * @param Request $request
+     * @return RedirectResponse
+     */
+    public function store(Request $request)
+    {
+        $validated = $request->validate([
+            "question" => "required",
+            "answer" => "required"
+        ]);
+        FAQ::create([
+            "question" => $validated["question"],
+            "answer" => $this->purifier->purify($validated["answer"])
```

Comment:
Nice that you added some extra security and neatness here, but the way you implemented is makes it harder for developers to implement it in for example the PostController. You see the code duplication there. In general, Laravel already provide a lot of security features for handling user input. But if you really feel that you need extra protection, try to find a solution that can be used while handling any request. For example, there might be a [library ](https://github.com/mewebstudio/Purifier) for this. Also, consider implementing an mutator for this.